core/internal/gethwrappers/generation/compile_command.txt

```diff
@@ -0,0 +1 @@
+yarn && myarn workspace @chainlink/contracts belt compile solc
```

Comment:
What is `myarn`? Typo? If so, is this file being used?

Reply:
Yes, it was a typo, and yes, this file is used. Thanks for catching it.